enjoy.py

```diff
@@ -19,18 +19,46 @@ def main():  # noqa: C901
     parser = argparse.ArgumentParser()
     parser.add_argument("--env", help="environment ID", type=str, default="CartPole-v1")
     parser.add_argument("-f", "--folder", help="Log folder", type=str, default="rl-trained-agents")
-    parser.add_argument("--algo", help="RL Algorithm", default="ppo", type=str, required=False, choices=list(ALGOS.keys()))
+    parser.add_argument(
```

Comment:
Could you revert the formatting that is not due to your PR?
(please use latest version of black with `-l 127` as specified in the contributing guide)